README.md

```diff
@@ -207,11 +208,11 @@ Just specify a GitHub repository in the URL and off you go!
 - [ ] add GitHub search
 
 
-[gh-board-image]: https://img.shields.io/github/issues/philschatz/gh-board.svg?label=Issues%20%28gh-board%29
+[gh-board-image]: https://img.shields.io/github/issues/coala/gh-board.svg?label=Issues%20%28gh-board%29
 [gh-board-url]: http://philschatz.com/gh-board/
```

Comment:
Change this. 
hint: I already said once on https://github.com/coala/gh-board/pull/176#discussion_r228795474